tt_metal/llrt/watcher.cpp

```diff
@@ -620,6 +671,23 @@ void watcher_init(int device_id,
         }
     }
 
+    // Initialize ethernet cores debug values
+    for (const CoreCoord &eth_core : ethernet_cores()) {
```

Comment:
Is this looping over all ethernet cores? I think it is ok for now but Umair is putting some custom dispatch kernels to inactive cores. Please sync with him to make sure he's aware that the eth mailbox address is going to be written to.

Reply:
Will do!

Reply:
Correction: this is active ethernet cores only - watcher will only poll active ethernet cores